osg/short-term.md

```diff
@@ -169,37 +175,23 @@ tool.
 
 ## Initializing the iRODS Connection
 
-Initializing the iRODS connection is actually fairly simple if you're using version 4.x of the icommands, which is
-highly recommended. The username in the `irods_user` configuration setting is guaranteed to have an empty password, so
-iRODS can be initialized by creating a file called `$HOME/.irods/irods_environment.json`. This file should be in the
-following format:
-
-```
-{
-    "irods_user_name": "<username>",
-    "irods_host": "<host-name>",
-    "irods_port": <port-number>,
-    "irods_zone_name": ""
-}
-```
-
-For the time being, the zone name should always be equal to the empty string. The rest of the settings are extracted
-from the job configuration file. After creating this file, `iget` and `iput` can be used in conjunction with the iRODS
-tickets to retrieve files from or push files to iRODS.
+As long as you're using version 4.x of the icommands and not using custom iRODS credentials, initializing the iRODS
+connection is very easy. The file, `config.json` is made to be compatible with the file that the icommands use to
```

Comment:
~I'd either drop the comma after file or add another comma after config.json.~ Just kidding, I see there's another file being referred to, so I'd lop off the leading couple words:
```suggestion
connection is very easy. `config.json` is made to be compatible with the file that the icommands use to
```